docs/csharp/language-reference/compiler-options/listed-alphabetically.md

```diff
@@ -59,7 +59,7 @@ The following compiler options are sorted alphabetically. For a categorical list
 |**/incremental**|Enables incremental compilation [obsolete].|  
 |[/keycontainer](../../../csharp/language-reference/compiler-options/keycontainer-compiler-option.md)|Specifies a strong name key container.|  
 |[/keyfile](../../../csharp/language-reference/compiler-options/keyfile-compiler-option.md)|Specifies a strong name key file.|  
-|[/langversion:\<string>](../../../csharp/language-reference/compiler-options/langversion-compiler-option.md)|Specify language version mode: ISO-1, ISO-2, 3, 4, 5, 6, or Default|  
+|[/langversion:\<string>](../../../csharp/language-reference/compiler-options/langversion-compiler-option.md)|Specify language version mode: ISO-1, ISO-2, 3, 4, 5, or Default|  
```

Comment:
> I just installed vs build tools 2017, and officially it has no 7 option, just 6 and latest. It does silently support 7, but it's not listed in it's /? notes

Yes, but I believe that's a bug in the description. Here is a PR that adds 7 and 7.1, which will be included in the next version of the compiler: https://github.com/dotnet/roslyn/pull/19113.